packages/lib/src/utils/FileSystem.ts

```diff
@@ -0,0 +1,111 @@
+import * as fs from 'fs'
+import * as path from 'path'
+
+// TS-TODO: Analyze which of these functions could be encapsulated.
+// TS-TODO: Make this a class with static methods?
```

Comment:
Please don't! It's perfectly ok for plain functions to be _just functions_, no need to wrap them in a class, object, or anything.